### PR TITLE
Add memory-optimized primerclip as a separate process

### PR DESCRIPTION
**Pull request by jkokosar**

### CODE REVIEW

File: resolwe_bio/processes/amplicon/bwa_trim.yml [hunk 1]
```diff
@@ -9,11 +9,11 @@
     expression-engine: jinja
     executor:
       docker:
-        image: resolwebio/dnaseq:1.1.0
+        image: resolwebio/legacy:latest
     resources:
       memory: 32768
   data_name: "Alignment ({{ reads|sample_name|default('?') }})"
-  version: 0.1.0
+  version: 0.0.8
```
Comment (mstajdohar) [inline resolwe_bio/processes/amplicon/bwa_trim.yml]: The version should be higher then 0.1.0

File: resolwe_bio/processes/amplicon/bwa_trim.yml [hunk 2]
```diff
@@ -193,7 +193,7 @@
       echo "trimming primers"
       primerclip \
         "${NAME}_align_unsorted.sam" \
-        {{master_file.master_file.file}} \
+        {{master_file.primers.file}} \
```
Comment (mstajdohar) [inline resolwe_bio/processes/amplicon/bwa_trim.yml]: Will this work with the new masterfile?

File: resolwe_bio/processes/amplicon/bwa_trim.yml (hunk 1 above)
Comment (jkokosar) [reply resolwe_bio/processes/amplicon/bwa_trim.yml]: The process with version 0.1.0 was removed from the database, the current one is 0.0.7, right?

File: resolwe_bio/processes/amplicon/bwa_trim.yml (hunk 2 above)
Comment (jkokosar) [reply resolwe_bio/processes/amplicon/bwa_trim.yml]: This is legacy process, for the old primerclip

File: resolwe_bio/processes/amplicon/bwa_trim.yml (hunk 1 above)
Comment (mstajdohar) [reply resolwe_bio/processes/amplicon/bwa_trim.yml]: Not on all deploys.

**Comment by jkokosar** [comment]
Closed in favour of https://github.com/genialis/resolwe-bio/pull/432

**Pull request closed without merging**